### PR TITLE
Add material for ENGAGE capacity building workshop

### PR DESCRIPTION
**Pull request by danielhuppmann**
This PR adds the links for the ENGAGE capacity building material.

### CODE REVIEW

File: tutorials/pyam.rst [hunk 1]
```diff
@@ -1,12 +1,26 @@
...
+ENGAGE Capacity Building Workshop (February 1, 2022)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+At a capacity-building workshop on good modeling practices
+organized by the `Horizon 2020 project ENGAGE`_ on February 1, 2022,
+`Daniel Huppmann <https://www.iiasa.ac.at/staff/daniel-huppmann>`_ and
+`Philip Hackstock <https://iiasa.ac.at/staff/philip-hackstock>`_
+presented an overview and live demo of the **pyam** package.
+
+ - Presentation slides: https://pure.iiasa.ac.at/id/eprint/17783/
```
Comment (LauWien) [inline tutorials/pyam.rst]:
```suggestion
 - Presentation slides (english): https://pure.iiasa.ac.at/id/eprint/17783
```

Maybe we can also use the following link for the presentation slides for "Presentation at the Strommarkttreffen (July 14, 2021)" http://pure.iiasa.ac.at/id/eprint/17319 to have it in the same layout. E.g.:

- Presentation slides (german/english): http://pure.iiasa.ac.at/id/eprint/17319

File: tutorials/pyam.rst [hunk 2]
```diff
@@ -1,12 +1,26 @@
...
+ - Tutorial notebook: https://github.com/danielhuppmann/ENGAGE-pyam-tutorial
```
Comment (LauWien) [inline tutorials/pyam.rst:20]:
```suggestion
 - Tutorial notebook (english): https://github.com/danielhuppmann/ENGAGE-pyam-tutorial
```

**Review by LauWien — COMMENTED** [review]
Looks good, thank you for adding! Just a few layout suggestions in line.

**Comment by danielhuppmann** [comment]
Thanks @LauWien - I changed the link for the Strommarkttreffen to point to pure directly instead of showing the DOI.

About the languages, I think that it should be clear that by default all material here is in English - and we only add the language if it is ambiguous (which it is because the Strommarkttreffen is a German-language event).

**Comment by LauWien** [comment]
> About the languages, I think that it should be clear that by default all material here is in English - and we only add the language if it is ambiguous (which it is because the Strommarkttreffen is a German-language event).

Makes sense! We could then think about also deleting "(english)" in the bullet point of "Tutorial notebook", but also good as is.

**Pull request merged**